packages/backend/src/services/CsvService/CsvService.ts

```diff
@@ -283,9 +284,13 @@ export class CsvService extends BaseService {
         const fileId = CsvService.generateFileId(fileName, truncated);
         const writeStream = fs.createWriteStream(`/tmp/${fileId}`);
 
-        const sortedFieldIds = Object.keys(rows[0])
-            .filter((id) => selectedFieldIds.includes(id))
-            .sort((a, b) => columnOrder.indexOf(a) - columnOrder.indexOf(b));
+        const sortedFieldIds = isNil(rows[0])
```

Comment:
just curious, what is the advantage of using isNil ? (Checks if value is null or undefined)
Is this not the same as doing `rows[0] ? []: ... `

Reply:
You're right! in this case there's not much difference

I use `isNil` or `!= null` to check for `null | undefined` exclusively, `rows [0] ? ...` would also return false for empty string, zero and `false`